www/privoxy/Portfile

```diff
@@ -58,16 +58,31 @@ worksrcdir          ${name}-${version}-stable
 depends_build-append \
                     port:p${perl5.major}-sgmlspm
 
-depends_lib-append \
-                    port:brotli \
+depends_lib-append  port:brotli \
                     port:pcre \
                     port:zlib
 
+depends_run-append  port:perl${perl5.major} \
+                    port:p${perl5.major}-getopt-long \
+                    port:p${perl5.major}-time-local
+
 set privoxyGroup    privoxy
 set privoxyUser     privoxy
 add_users           ${privoxyUser} group=${privoxyGroup}
 
-# diff -Naur privoxy-orig/config privoxy-new/config | sed -E -e 's/privoxy-(orig|new)/./g'
+# privoxy patch file creation
+## export prefix=${prefix:-/opt/local}
+## mkdir privoxy-orig privoxy-new
+## sudo cp ${prefix}/etc/privoxy/config.new privoxy-orig/config
+## sudo cp ${prefix}/etc/privoxy/match-all.action.new privoxy-orig/match-all.action
+## sudo cp ./privoxy-orig/config ./privoxy-new/
+## sudo cp ./privoxy-orig/match-all.action ./privoxy-new/
+## sudo chown `whoami` privoxy-orig/config privoxy-new/config privoxy-orig/match-all.action privoxy-new/match-all.action
+## patch -p0 -f -l -N privoxy-new/config < ${prefix}/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/www/privoxy/files/patch-config.diff
+## patch -p0 -f -l -N privoxy-new/match-all.action < ${prefix}/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/www/privoxy/files/patch-match-all.action.diff
+## diff -NaurdwB ./privoxy-orig/config ./privoxy-new/config | sed -E -e 's/\.\/privoxy-(orig|new)\/(config)(\.[[:alnum:]]+)*/\.\/\2/' | sed -E -e 's|/opt/local|@@PREFIX@@|g' > ~/Downloads/patch-config.diff
+## diff -NaurdwB ./privoxy-orig/match-all.action ./privoxy-new/match-all.action | sed -E -e 's/\.\/privoxy-(orig|new)\/(match-all\.action)(\.[[:alnum:]]+)*/\.\/\2/' | sed -E -e 's|/opt/local|@@PREFIX@@|g' > ~/Downloads/patch-match-all.action.diff
+
```

Comment:
What's the purpose of this big block of commented out lines?

Reply:
The next time I or someone needs to generate a patch from the upstream’s configuration files, those are the commands that accomplish it. Easier than doing all that from scratch.

Reply:
Then the comment should explain that this is what is being described. You should have a description saying "this is how you generate a patch" or whatever would otherwise give someone in the future enough information to follow in your footsteps. As it stands, the commands appear in comments without explanation.

Reply:
Thanks, done.